Automattic/jetpack
# Map block: Set showsMapTypeControl to false

Fixes https://github.com/Automattic/wp-calypso/issues/76288

## Proposed changes:
This PR hides the map type control.

| Before | After |
|-|-|
| <img width="409" alt="CleanShot 2023-05-05 at 10 00 50@2x" src="https://user-images.githubusercontent.com/528287/236405931-2cfec42d-b7ec-41a9-8f4b-243a01ccd74b.png"> | <img width="311" alt="CleanShot 2023-05-05 at 10 01 21@2x" src="https://user-images.githubusercontent.com/528287/236405961-83a3966a-e7ad-4908-aed5-c17e7fb5a664.png"> |

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
1. Apply this PR (either by running Jetpack locally or using the JP downloader to your sandbox)
2. If running locally enable Mapkit by setting the cookie (`document.cookie = 'map_provider=mapkit';`)
3. Add a new post and add a map
4. Confirm that the map type control doesn't show up in both front- & backend.
